src/game/interactables_2/sky_canyon/propeller_spring.c

```diff
@@ -58,13 +59,9 @@ static void sub_807C2E0(void)
             r1 = 6;
         }
     } else {
-#ifndef NON_MATCHING
```

Comment:
Why remove this?

Reply:
I mean, it's less code.
And sure, the do-while isn't pretty, but since it compiles to a matching version, why would we change it?
Or do you think it's less optimizable for other compilers to have the do-while around?

Reply:
Fixed, btw. ^^